qiita_ware/processing_pipeline.py

```diff
@@ -22,51 +22,56 @@
 
 def _get_qiime_minimal_mapping(prep_template, out_dir):
     """Generates a minimal QIIME-compliant mapping file for split libraries
-
     The columns of the generated file are, in order: SampleID, BarcodeSequence,
```

Comment:
Yes it is, is not required but we should use it if it exists in the mapping file.